docs/migration-guides/MIGRATION_GUIDE_v5_to_v6.md

```diff
@@ -218,6 +222,12 @@ S3 client.
 Additionally, you need to annotate your S3 tests with `@LocalS3` due to an implementation detail
 of the underlying test library.
 
+### 11 sda-commons-server-prometheus
+
+The endpoint `/healthcheck/prometheus` was removed. Health checks metrics are available using endpoint `/metrics/prometheus`.
+
+SDA specific metric `http_request_duration_seconds` was removed.
```

Comment:
It would be worth mentioning the alternative that you can use now.

Reply:
added